src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -1435,14 +1436,21 @@ def set_input(self, input_data):
         '''
         assert isinstance(input_data, AcquisitionData)
         self.input_data = input_data
+    def set_dcm_prefix(self, dcm_prefix):
+        self.dcm_prefix = dcm_prefix
     def process(self):
         '''
         Processes the input with the gadget chain.
+        dcm_prefix: Python text string.
+        If dcm_prefix is not None, the reconstructed images are written to
```

Comment:
actually at the moment you set it "", not to `None`. are both handled the same?

Reply:
just forgot to change None (used in my first design) to "", corrected now